tests/bofire/surrogates/test_diagnostics.py

```diff
@@ -205,6 +205,21 @@ def test_cvresult_get_metric_invalid():
         np.isnan(cv.get_metric(metric=metric))
 
 
+def test_cvresult_get_UQ_metric_valid():
+    cv = generate_cvresult(key="a", n_samples=10, include_standard_deviation=True)
+    assert cv.n_samples == 10
+    for metric in metrics.keys():
+        cv.get_metric(metric=metric)
+
+
+def test_cvresult_get_UQ_metric_invalid():
+    cv = generate_cvresult(key="a", n_samples=10, include_standard_deviation=False)
+    assert cv.n_samples == 10
+    for metric in metrics.keys():
```

Comment:
you have to iteratre here only over the uq metrices, as only there the warning is raised.